source/convergence_test.cc

```diff
@@ -413,8 +413,14 @@ Stream& operator<<(Stream &stream, const ConvergenceTestParameters &prm)
 ConvergenceTestData::ConvergenceTestData(const ConvergenceTestType &type)
 :
 type(type),
-level(0)
-{}
+level(0),
+dimension(0)
+{
+  table.declare_column("level");
+  table.declare_column("L2");
+  table.declare_column("H1");
+  table.declare_column("Linfty");
```

Comment:
Why did you chose to not add time step, global refinement level and maximum cirumradius?

Reply:
Declaring the columns is actually not necessary except for the level column because it should be printed as the first column.

Fixed in d423619.